__tests__/app.test.js

```diff
@@ -84,10 +77,45 @@ describe("GET /api/", () => {
 }) 
 
 
+describe("GET /api/articles", () => {
+    test("Check the request returns a 200 and has the correct properties", () => {
+
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+
+        expect(body.articles).toHaveLength(13)    
+        body.articles.forEach(article => {
+        expect(article).toHaveProperty('author', expect.any(String))
+        expect(article).toHaveProperty('title', expect.any(String))
+        expect(article).toHaveProperty('article_id', expect.any(Number))
+        expect(article).toHaveProperty('topic', expect.any(String))
+        expect(article).toHaveProperty('created_at', expect.any(String))
+        expect(article).toHaveProperty('votes', expect.any(Number))
+        expect(article).toHaveProperty('article_img_url', expect.any(String))
+        expect(article).toHaveProperty('commentCount', expect.any(Number))
+        expect(article).not.toHaveProperty('body')
+
+         })
 
+        })
 
+    })
 
+    test("Check the request returns in date descending order", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+
+        const descendingArray = [body.articles[0].created_at , body.articles[1].created_at, body.articles[2].created_at]
```

Comment:
You've put these in order manually - so they're definitely in order. You should instead be actually checking that the body that returns is in descending order.